sources/packages/backend/apps/api/src/app.module.ts

```diff
@@ -120,6 +122,8 @@ import { DatabaseModule } from "@sims/sims-db";
     DesignationAgreementLocationService,
     StudentService,
     SFASIndividualService,
+    SFASApplicationService,
```

Comment:
The reason that we need to add these services here is that we have too much left over in the `app.module` and others.
For instance, I did a cleanup in the `app.module` `providers` and it can look like below.
```ts
controllers: [
    AppController,
    UserController,
    ProgramYearController,
    ConfigController,
    DynamicFormController,
  ],
  providers: [
    AppService,
    UserService,
    BCeIDServiceProvider,
    FormService,
    ProgramYearService,
  ],
```
Maybe @andrepestana-aot can take care of it in his current API cleanup ticket.
@ann-aot @guru-aot FYI

Reply:
So if I take update after your changes are merged, I may not require to add SFASApplicationService to app module right?

Reply:
I did not change it on the repo, I just did the test while I was with your branch on my local.
The first list of the providers (the short one), is all that is needed. If you copy and paste that into your code it is supposed to just work. I removed the dependency and the dependency of the dependency and so on 😉